### PR TITLE
Rainier 4U DDR5 Odyssey support & minor update to Rainier 2U

### PR DESCRIPTION
**Pull request by ruby452**
1.) First support of Odyssey DDR5 4U DDIMMs for Rainier 4U MRW
2.) Minor update to Rainier 2U MRW to make TYPE=TEMP_SENSOR for the 3 temp sensors under the OCMB

### CODE REVIEW

**Comment by dcrowell77** [comment]
The 4U contents seem correct to me.  Both are still not producing temp sensors for the DDIMMs, but that is something we can live with for now (no change from current state of affairs on Rainier-2U).

**Comment by ruby452** [comment]
Reverted TYPE of TEMP_SENSOR to NA since it's causing problems with the code

File: Rainier-2U-MRW.xml [hunk 1]
```diff
@@ -130018,7 +130018,7 @@
 	</attribute>
 	<attribute>
 		<id>TYPE</id>
-		<default>TEMP_SENSOR</default>
+		<default>NA</default>
```
Comment (dcrowell77) [inline Rainier-2U-MRW.xml]: The TEMP_SENSOR TYPE was not the problem, the <type>unit-i2c-slave</type> is.

**Comment by dcrowell77** [comment]
The HW CI failure was on Everest and was just a known issue.  I restarted it to get things to pass clean.  Assuming it does, we can merge this as-is without the temp_sensor stuff since it should get the 4U DDIMMs into the same state as the 2Us are now.

**Pull request merged**